install_workflow/nurion_python3_requirements.txt

```diff
@@ -0,0 +1,74 @@
+absl-py==0.6.1
```

Comment:
We'll need to nuke down the lists. We have way too much version incompatibility with the defaults on maui. And we shouldn't copy our mess.

Reply:
@jpa198 This one too actually.